File: service/Api/QuizRunStep/Timer/TimerStep.cs
namespace Quizmeister
{
    public class TimerStep
    {
        public int QuestionId { get; set; }
        public int QuizId { get; set; }
        public int Total { get; set; }
        public int Current { get; set; }
        public int Percentage { get; set; }
    }
}

File: app/projects/domain/src/lib/quiz-run-steps/timer/timer-step.ts
import { MessageStatus } from '@domain/infrastructure/message-status';

export interface TimerStepEvent {
  step: TimerStep;
  status: MessageStatus;
  message: string;
}

export interface TimerStep {
  questionId: number;
  quizId: number;
  total: number;
  current: number;
  percentage: number;
}
